substrate/utils/frame/benchmarking-cli/src/machine/reference_hardware.json

```diff
@@ -3,6 +3,11 @@
 		"metric": "Blake2256",
 		"minimum": 1000.00
 	},
+	{
+		"metric": {"Blake2256Parallel":{"num_cores":8}},
+		"minimum": 1000.00,
+		"check_on_rc_authority": true
```

Comment:
Maybe a better name would be `validator_only` instead of `check_on_rc_authority`?

Reply:
Done.